osi_sensordata.proto

```diff
@@ -60,143 +68,137 @@ message SensorData
+    // the sensor-relative coordinates in detected objects of the sensor
+    // as a whole.  Individual features detected by individual physical
+    // detectors are governed by the actual physical mounting positions
+    // of the detectors, as indicated in the technology-specific sub-views
+    // and sub-view configurations.
     //
     // \arg \b x-direction of sensor coordinate system: sensor viewing direction
     // \arg \b z-direction of sensor coordinate system: sensor (up)
     // \arg \b y-direction of sensor coordinate system: perpendicular to x and z right hand system
     //
-    // \par References: 
+    // \par References:
     // [1] DIN ISO 8855:2013-11
     //
     // \note This field is usually static during the simulation.
-    // \note The origin of vehicle's coordinate system in world frame is 
+    // \note The origin of vehicle's coordinate system in world frame is
     // ( \c Vehicle::base.position + Inverse_Rotation_yaw_pitch_roll(\c Vehicle::base.orientation) * \c Vehicle::bbcenter_to_rear) .
-    // The orientation of the vehicle's coordinate system is equal to the orientation 
+    // The orientation of the vehicle's coordinate system is equal to the orientation
     // of the vehicle's bounding box \c Vehicle::base.orientation.
     //
-    optional MountingPosition mounting_position = 6;
+    optional MountingPosition mounting_position = 4;
 
     // The root mean squared error of the mounting position.
     //
-    optional MountingPosition mounting_position_rmse = 7;
+    optional MountingPosition mounting_position_rmse = 5;
+
+    // Sensor view w.r.t. the sensor coordinate system
+    //
+    // This provides a copy of the \c SensorView data received by the sensor
+    // for reference purposes.
+    //
+    optional SensorView sensor_view = 6;
 
     // The timestamp of the last real-world measurement (e.g. GT input) that
     // this set of sensor data takes into account.  This in effect is the last
     // time instance of reality the measurements correspond to.  See field
     // \c SensorData::timestamp for a detailed discussion.  This value is also
     // the upper bound to the \c DetectedObjectHeader::measurement_time and the
     // feature data \c SensorDetectionHeader::measurement_time fields.
-    // 
-    optional Timestamp last_measurement_time = 8;
+    //
+    optional Timestamp last_measurement_time = 7;
 
     // General information about the \c DetectedObject .
-    // 
-    optional DetectedEntityHeader object_header = 100;
-    
+    //
+    optional DetectedEntityHeader object_header = 8;
+
     // The list of objects detected by the sensor as perceived by the sensor.
     //
     repeated DetectedObject object = 9;
-    
+
     // General information about the \c DetectedTrafficSign .
-    // 
-    optional DetectedEntityHeader traffic_sign_header = 101;
+    //
+    optional DetectedEntityHeader traffic_sign_header = 10;
```

Comment:
I didn't touch the naming stuff, since I think that should be a seperate PR so as not to confuse everyone (I've included the renumberings, because that is a more pervasive problem, and I had to renumber in any case). I'm not that happy with the whole DetectedEntityHeader field naming, and semantics, since it sounds like a header for each detected entity, whereas it really is only a header for each class of entities...

Reply:
@CarloVanDriestenBMW I would leave traffic_sign and traffic_sign_header. Otherwise we have to add the whole scope of a field to all field names. 
@pmai Only DetectedHeader?

Reply:
@carsten-kuebler, @CarloVanDriestenBMW  I'm thinking of something like DetectedEntityClassHeader and traffic_sign_class_header that makes the per-class character clearer, but those names are rather unwieldy as well, so I'm inclined to leave this as is for now...

Reply:
If we don't use a message Message, we could use Message as a fixed expression.
DetectedMessage(s)Header